src/wattsi.pas

```diff
@@ -1225,8 +1281,24 @@    TCrossReferences = record
          {$IFDEF DEBUG} Writeln('Finding IDs and stripping excluded sections...'); {$ENDIF}
          FirstPass();
          // Second pass - make the changes to the DOM that we need
-         {$IFDEF DEBUG} Writeln('Adjusting headers, references, finding cross-references...'); {$ENDIF}
-         SecondPass();
+         if (Variant = vDEV) then
+         begin
+            {$IFDEF DEBUG} Writeln('Adjusting headers, references, finding cross-references, creating search-index.json...'); {$ENDIF}
+            Assign(SearchIndexJsonFile, OutputDirectory + '/multipage-dev/search-index.json');
+            Rewrite(SearchIndexJsonFile);
+            Write(SearchIndexJsonFile, '[');
+            SecondPass();
+            try
```

Comment:
> What is this try/except about?

I found that without that I’d get an exception there about the file being closed already. I’m not sure why but the easiest thing to do was just to ignore the exception.